galsim/interpolatedimage.py

```diff
@@ -25,6 +25,7 @@
 from . import _galsim
 from ._galsim import Interpolant
 from ._galsim import Nearest, Linear, Cubic, Quintic, Lanczos, SincInterpolant, Delta
+import numpy as np
```

Comment:
In any case, import numpy is pretty quick, so this probably isn't too important.  I suspect the one that is slowing us down is the `try: import astropy.io.fits; except: import pyfits` bit in our `__init__.py` file.  That one takes a bit longer.  I should see if I can find a way to delay that until we need it.